### PR TITLE
Enable configuring ntp server pool in ntp_global

### LINKED ISSUE #221: [vyos_ntp_global] `dynamic` isn't a valid vyos option for a server, and `pool` is missing
##### SUMMARY

#175 added `vyos_ntp_global`, which allows setting `options` on `servers`.  Per the documentation, those options are `noselect`, `dynamic`, `preempt`, and `prefer`.  However, documentation for [equuleus](https://docs.vyos.io/en/equuleus/configuration/system/ntp.html) and [dev](https://docs.vyos.io/en/latest/configuration/system/ntp.html) do not have `dynamic`, but instead `pool` ([crux](https://docs.vyos.io/en/crux/configuration/system/ntp.html) documentation is lacking clarity).  `dynamic` is not a supported option for vyos, so it's impossible to set a pool with this module.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

`vyos.vyos.vyos_ntp_global`

##### STEPS TO REPRODUCE

```yaml
    - vyos.vyos.vyos_ntp_global:
        config:
          servers:
            - server: "0.pool.ntp.org"
              options:
                - "dynamic"
```

##### EXPECTED RESULTS

All documented options to work, and all options to be allowed for a server.  I would find it acceptable if `dynamic` maps to `pool` if this module is trying to maintain compatibility with out networking modules, but that should be documented as well.

##### ACTUAL RESULTS

If you use the above example, you get an error (tested on equuleus):
```
Configuration path: system ntp server 0.pool.ntp.org [dynamic] is not valid
```

If you try to manually specify `pool` an an option, you get an error:
```
value of options must be one or more of: noselect, dynamic, preempt, prefer. Got no match for: pool found in config -> servers
```

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Fixes #221 

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: plugins/module_utils/network/vyos/argspec/ntp_global/ntp_global.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ class Ntp_globalArgs(object):  # pylint: disable=R0903
                                 "dynamic",
                                 "preempt",
                                 "prefer",
+                                "pool",
```
Comment (sdwilsh) [inline plugins/module_utils/network/vyos/argspec/ntp_global/ntp_global.py:52]: nit: this list of options was alphabetized prior to this change. This applies elsewhere in this change as well.

**Review by sdwilsh — APPROVED** [review]

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**